starters/angular/basic/apphosting.yaml

```diff
@@ -0,0 +1,23 @@
+# Settings for Backend (on Cloud Run).
```

Comment:
Maybe we should have a header explaining the use of environment-specific apphosting.yaml overrides?

Reply:
I think I'd rather point the users to public doc so we don't have 2 separate places to track documentations on how to use env specific overrides.